addons/website/static/src/snippets/s_share/000.js

```diff
@@ -59,9 +59,13 @@ const ShareWidget = publicWidget.Widget.extend({
                 // parameter, merging everything together, e.g of output:
                 // https://wa.me/?text=%20OpenWood%20Collection%20Online%20Reveal%20%7C%20My%20Website%20http%3A%2F%2Flocalhost%3A8888%2Fevent%2Fopenwood-collection-online-reveal-2021-06-21-2021-06-23-8%2Fregister
                 // For more details, see https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/
-                modifiedUrl.searchParams.set(titleParamFound, `${currentTitle}%20${currentUrl}`);
+                modifiedUrl.searchParams.set(titleParamFound, `${currentTitle} ${currentUrl}`);
             } else {
-                modifiedUrl.searchParams.set(titleParamFound, currentTitle);
+                // The built-in `URLSearchParams.set()` method encodes spaces
+                // as "+" characters, which are not properly parsed as spaces
+                // by email clients, so we can't use it here.
+                modifiedUrl.search = modifiedUrl.search
+                    .replace(encodeURIComponent("{title}"), encodeURIComponent(currentTitle));
```

Comment:
Questions about that: 
I am not sure but isn't the issue you are fixing in fact a mail client issue ? Like, it should be their job to better "parse" the URL in their app, no ? And so we shouldn't do anything on our side, since everything is properly encoded with `searchParams.set`, no ? :thinking: 

Also, is the issue happening with every mail client ever ? Or only with the one you have on your computer ? I personally have "Geary" on my computer (I don't use it, I use gmail in my browser but it is the one opening when sharing by mail) and it indeed have the "+" in the mail subject but I can't be sure for other mail clients, did you test with other ones ? :thinking:

Reply:
Good question :D Obviously I can't answer for all mail clients, but I tested on Geary, Thunderbird and on mobile on K9 Mail (but that's kind of the equivalent of Thunderbird on mobile, so that would be logical). I just asked ARD and it's also the case on the default Apple client.
I tried to test on Outlook on someone else's computer, but the client never launched, so I don't know.

As it's the case on the Apple client and Thunderbird, I think that's reason enough - even if we think that should be the client's job to do it. But if you search a bit the issue on Google, there have been a few questions around it on several mail clients over the years. I doubt it's been fixed on most.

Reply:
On second thought, I'm not even sure that's a bug on the clients' side. That's probably deliberate. It makes sense in actual URLs that spaces and + signs are equivalent; but in email subjects, there's no reason you shouldn't be able to use both spaces and + signs in a subject line - so the logical consequence is that mail clients cannot parse an encoded + sign as a space.

Reply:
Okay then I guess it can stay like that 
since it was already `encodeURIComponent` before the refactoring, I see nothing wrong with setting it like that again :slightly_smiling_face:

Reply:
:+1: I'll update the commit message.